provisioner/docker/docker-hadoop.sh

```diff
@@ -271,7 +271,11 @@ env-check() {
     echo "Check docker:"
     docker -v || exit 1
     echo "Check docker-compose:"
-    $DOCKER_COMPOSE_CMD -v || exit 1
+	if [ "$DOCKER_COMPOSE_CMD" == "docker-compose" ]; then
+      $DOCKER_COMPOSE_CMD -v || exit 1
+	elif [ "$DOCKER_COMPOSE_CMD" == "docker compose" ]; then
+      $DOCKER_COMPOSE_CMD version || exit 1
+	fi 
```

Comment:
@yamasakisua basically LGTM. 

- Could you replace tabs with whitespaces? 
- Could you remove a trailing whitespace in line 278?